.github/dependabot.yml

```
@@ -9,3 +9,6 @@ updates:
     directory: /
     schedule:
       interval: daily
+    ignore:
+      - dependency-name: '@types/*'
```

Should we be explicit about the name (e.g.)

```suggestion
      - dependency-name: '@types/node'
```

or is there a reason to use the wildcard here?

---

To prevent making unexpected different major versions between implementation and DefinitelyTyped. Current `package.json` does not depend on another DefinitelyTyped. However https://github.com/actions/dependency-review-action/pull/131 is integrating a new one.
As far as I know, this wildcard is useful when you only want minor and patch PRs from dependabot . Of course I can update this to focus only on the `node` issue.